addons/mail/static/src/components/rtc_activity_notice/rtc_activity_notice.xml

```diff
@@ -6,16 +6,19 @@
             <t t-if="messaging">
                 <RtcInvitations/>
                 <t t-if="messaging.rtc.channel">
-                    <a class="o_RtcActivityNotice_button px-3 user-select-none dropdown-toggle o-no-caret o-dropdown--narrow" t-att-title="rtcActivityNoticeView.buttonTitle" role="button" t-on-click="messaging.rtc.onClickActivityNoticeButton">
+                    <button class="o_RtcActivityNotice_button px-3 user-select-none dropdown-toggle o-no-caret o-dropdown--narrow" t-att-title="rtcActivityNoticeView.buttonTitle" role="button" t-on-click="messaging.rtc.onClickActivityNoticeButton">
                         <div class="o_RtcActivityNotice_buttonContent d-flex align-items-center">
-                            <i class="o_RtcActivityNotice_outputIndicator fa fa-lg" t-att-class="{
-                                'fa-microphone': !messaging.rtc.sendDisplay and !messaging.rtc.sendUserVideo,
-                                'fa-video-camera': messaging.rtc.sendUserVideo,
-                                'fa-desktop': messaging.rtc.sendDisplay,
-                            }"/>
+                            <span class="fa-stack me-1">
+                                <i class="fa fa-stack-2x fa-square text-danger"/>
```

Comment:
Visual change:
**before**
<img width="176" alt="before" src="https://user-images.githubusercontent.com/6569390/168621647-f67e00a8-af47-4bd4-8672-1578c2ddc017.png">
**after**
<img width="189" alt="after" src="https://user-images.githubusercontent.com/6569390/168621680-a9180879-eb8a-46f8-a078-8c04420fc31f.png">

I don't think this just increases the visibility of the button. It's really a design choice, so definitely out of scope of this PR.

IMO this design has some problems, like colour being too aggressive (red), reduce visibility of the mic icon, and clashing colour/visual in comparison to other systray items. The icon part is more catchy, so there are some good things, but it's not good enough of a change for me.

I think we can take some inspiration with what macOS does in the menu bar as a feedback that the mic is being used. The choice of colour is less aggressive, and it's less catchy while still visible at a quick glance (see orange dot):
<img width="206" alt="mic-in-use-macos" src="https://user-images.githubusercontent.com/6569390/168623875-9aa551c7-fcd0-41b6-9477-5425fb002a62.png">

Reply:
@alexkuhn 👋 
I updated the design, what do you think ? 🙂 

https://user-images.githubusercontent.com/80679690/169481489-e3942096-5ef9-4796-aa6f-e850b3e52a4a.mov

Should I move it in another PR ?

Reply:
It looks quite good!

I'm not entirely sure we want the blinking animation... But it's fine to keep it for now, because it's barely distracting.

------

I think the reason the blinking is undesirable is because it distracts with its animation. Sometimes it makes sense, like when you record something: it lets you know that it's being recording, and it's a way to keep you inform to stop recording when you finished your work (because forgetting means wasted CPU/memory/etc.).

**Having active mic is not the same as recording,** and potentially having mic active can be there 100% of the time using odoo. So it can be very annoying to have a distraction 100% of the time.

Actually many POCs before merging this feature were flavours of blinking effects, but were judged too distracting, hence we picked a static visual.

Reply:
> Should I move it in another PR ?

This can be kept in this PR, in its own commit, like how it's done now. Thanks!